runtime/api/src/main/java/org/qbicc/runtime/CNative.java

```diff
@@ -937,6 +938,7 @@ public void set(int arrayIdx, T newVal) {
         public native F getInvokable();
     }
 
+    public static final class void_ptr_unaryoperator_function_ptr extends ptr<function<UnaryOperator<void_ptr>>> {}
```

Comment:
Ohh, yeah I didn't think about how reifying the pointer types would affect function pointers.  I will probably have to think about that some more.

Reply:
Yea there may be a lot of these eventually. Wasn't this format intended to be temporary?

Reply:
Yeah, I'd really like to go back to inferring them since it's a much nicer experience.